client/marshaller/kryo-marshaller/src/main/java/org/infinispan/marshaller/kryo/KryoMarshaller.java

```diff
@@ -0,0 +1,57 @@
+package org.infinispan.marshaller.kryo;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import org.infinispan.commons.io.ByteBuffer;
+import org.infinispan.commons.io.ByteBufferImpl;
+import org.infinispan.commons.marshall.AbstractMarshaller;
+import org.infinispan.commons.marshall.Marshaller;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.0
+ */
+public class KryoMarshaller extends AbstractMarshaller {
+
+   private final Kryo kryo = new Kryo();
+
+   public Kryo getKryo() {
+      return kryo;
+   }
+
+   @Override
+   public Object objectFromByteBuffer(byte[] bytes, int offset, int length) throws IOException, ClassNotFoundException {
```

Comment:
The `throws` are not needed unless the implementation actually throws them. Don't think it's the case here?